server/views/pages/caseSummary.njk

```diff
@@ -68,6 +68,23 @@
                     {% endif %}
                 </div>
             </div>
+            {% if flags.flagCancelRecommendationProcess and  recommendationBanner.display %}
+                
+                {% set bannerHtml %}
+                    <p>{{ recommendationBanner.createdByUserFullName }} {{ recommendationBanner.text}} {{ recommendationBanner.personOnProbationName }} on {{ formatDateTimeFromIsoString({ isoDate: recommendationBanner.createdDate, dateOnly: true }) }}.</p>
+                    <p>
+                        <a href="">Cancel</a> if you need to stop this recommendation. You will be asked why.
+                    </p>
+                    </form>
```

Comment:
Looks like this form closing tag doesn't have an opening tag

Reply:
Removed